### ISSUE TITLE
:global() still gets component class added when ">" is not inside the :global()

### ISSUE THREAD

**Issue by FreekyMage** [sveltejs/svelte-preprocess#253]
**I posted this in the svelte repo first, because I think the fault is theirs, but I can't reproduce the error with pure svelte. The info below is what I posted there**

**Describe the bug**
When a `>` is behind a `:global()` and thus outside the parentheses, the globalised class still gets the class of the component.
I think it's a conflict between svelte and svelte-preprocess that got started with https://github.com/sveltejs/svelte/issues/4795 (we didn't have this issue yesterday)

**To Reproduce**
I couldn't reproduce the issue with https://svelte.dev/repl because I think it's a conflict between svelte and svelte-preprocess.

So in our project we have something like this:
```
:global(.menu) > .menu-item {...}
```
The result is the same with the normal style tag or one with `lang="scss"`.
This outputs to:
```
.svelte-100eekq.menu > .menu-item.svelte-100eekq {...}
```
The adjustment below does produce the expected output, but shouldn't be required I think:
```
:global(.menu >) .menu-item {...}
```
If this is ok for the svelte team, then this might need to be warned about in the documentation.

**Expected output**
```
.menu > .menu-item.svelte-100eekq {...}
```

**Information about your Svelte project:**
- node-sass: 4.14.1
- sass-loader: 8.0.2
- svelte 3.25.1
- svelte-preprocess: 3.9.12 (and 4.3.0 tested with same results)
- webpack 4.44.1

**Severity**
Blocking upgrade

**Comment by floratmin** [comment]
I have the same problem, but in a different context. When I add autoprefixer in svelte-preprocess, then I get `:global()` injected into my style imported from sass. I just created a repo for this:
https://github.com/floratmin/sapper-bulma-scss-treeshaking

**Comment by kaisermann** [comment]
Hey, @FreekyMage  👋  Would you be able to provide a minimal reproducible example? I'm unable to reproduce the issue on a test case:

![code](https://user-images.githubusercontent.com/12702016/94269883-a8d2ba00-ff15-11ea-8b5e-b269aa317dcf.png)

@floratmin Would you be able to provide a minimal repro? A sapper project with a lot of CSS makes it difficult to identify the issue! 😁

**Comment by floratmin** [comment]
@kaisermann Here is the minimal repro with svelte.
https://github.com/floratmin/svelte-autoprefixer-example

**Comment by kaisermann** [comment]
@floratmin Thanks for the repro I was able to pinpoint where the problem originated! Fixed in `4.4.2`. However, the OP issue still seems to be related to the compiler.
